root/MaptickAverager.py

```diff
@@ -242,6 +247,34 @@ def add_new_value(list, new_value, value_creator):
     player_total = 0
     td_total = 0
 
+    sendmaps_total_cost = 0 #all of these only use the last rows data which is the real total for that round
+    sendmaps_total_calls = 0
+
+    sendmaps_client_cost = 0
+    sendmaps_client_calls = 0
+
+    movable_changes_cost = 0
+    movable_changes_calls = 0
+
+    check_turf_vis_contents_cost = 0
+    check_turf_vis_contents_calls = 0
+
+    check_hud_image_vis_contents_cost = 0
+    check_hud_image_vis_contents_calls = 0
+
+    loop_turfs_in_range_cost = 0
+    loop_turfs_in_range_calls = 0
+
+    obj_changes_cost = 0
+    obj_changes_calls = 0
+
+    mob_changes_cost = 0
+    mob_changes_calls = 0
+
+    movables_examined = 0
```

Comment:
IDK why but this smells like something that could be done with a dictonary. Would be slower I think, but god this is

Reply:
didnt do the dict thing because it turned out to be way easier to just cut down the number of vars